minecraft-common/src/main/java/com/sigmundgranaas/forgero/minecraft/common/toolhandler/block/selector/RadiusVeinSelector.java

```diff
@@ -0,0 +1,86 @@
+package com.sigmundgranaas.forgero.minecraft.common.toolhandler.block.selector;
+
+import static com.sigmundgranaas.forgero.minecraft.common.toolhandler.block.selector.BlockSelectionUtils.around;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import org.jetbrains.annotations.NotNull;
+
+import net.minecraft.util.math.BlockPos;
+
+/**
+ * BlockSelector that selects chainedBlocks in a radius around the root position.
+ * <p>
+ * Will exclude blocks that are not valid, and try to find valid diagonal blocks.
+ *
+ * @author StevePlays
```

Comment:
Thanks for the credit here, but I'd appreciate it if you put `Steveplays28` instead to be consistent with the `fabric.mod.json` authors list.